Remotion/Data/DomainObjects.Web.Test/App.config

```diff
@@ -6,13 +6,22 @@
   <system.web>
     <pages>
       <controls>
+        <add tagPrefix="asp" namespace="System.Web.UI" assembly="CoreForms.Web.Extensions"/>
+        <add tagPrefix="asp" namespace="System.Web.UI.WebControls" assembly="CoreForms.Web.Extensions"/>
         <add tagPrefix="remotion" namespace="Remotion.ObjectBinding.Web.UI.Controls" assembly="Remotion.ObjectBinding.Web"/>
         <add tagPrefix="remotion" namespace="Remotion.Web.UI.Controls" assembly="Remotion.Web"/>
       </controls>
     </pages>
-    <compilation defaultLanguage="c#" debug="true" targetFramework="4.8" />
-    <httpRuntime requestValidationMode="2.0" targetFramework="4.8" />
+    <compilation defaultLanguage="c#" debug="true">
+      <assemblies>
+        <add assembly="System.Drawing.Primitives"/>
+      </assemblies>
+    </compilation>
+    <httpRuntime requestValidationMode="2.0"/>
     <authentication mode="Windows"/>
+    <httpModules>
+      <add name="Session" type="System.Web.SessionState.SessionStateModule"/>
+    </httpModules>
   </system.web>
   <system.webServer>
```

Comment:
`<system.WebServer>` should be obsolete. But we need to move the handlers over just like in the other PRs.